specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-10-01-preview/cancelEligibility.json

```diff
@@ -0,0 +1,180 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cancel Eligibility API",
+    "version": "2021-10-01-preview",
+    "description": "Cancel Eligibility API to check whether the subscription is eligible to cancel or not."
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "CancelEligibilityModel"
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/cancelEligibility": {
```

Comment:
>cancelEligibility [](http://example.com/codeflow?start=70&length=17)

This doesn't match requirements on verb usage.
It should be

POST checkCancelEligibility
or similar